From e8dae2d0a9d1a31daceb4a33cd49ba31779b20da Mon Sep 17 00:00:00 2001
From: Shunqian Zheng <zhengsq@rock-chips.com>
Date: Fri, 25 May 2018 14:05:19 +0800
Subject: [PATCH] ARM: configs: Build wifi drivers as modules for linux
 platform

Since the wireless drivers conflict when built-in together,
this makes the wireless build as modules for linux platform.

NOTE that this will affect all boards using
rockchip_linux_defconfig. But yes, it is a general defconfig,
we want to make it as common as possible.

Change-Id: If4fd669793b624a3721dd17822e5d90ca20e8b19
Signed-off-by: Shunqian Zheng <zhengsq@rock-chips.com>
---
 arch/arm/configs/rockchip_linux_defconfig | 11 ++++++-----
 1 file changed, 6 insertions(+), 5 deletions(-)

diff --git a/arch/arm/configs/rockchip_linux_defconfig b/arch/arm/configs/rockchip_linux_defconfig
index 0240a378355a..4cbedbc4b466 100644
--- a/arch/arm/configs/rockchip_linux_defconfig
+++ b/arch/arm/configs/rockchip_linux_defconfig
@@ -201,11 +201,12 @@ CONFIG_HOSTAP_FIRMWARE_NVRAM=y
 CONFIG_RT2X00=y
 CONFIG_RT2800USB=y
 CONFIG_WL_ROCKCHIP=y
-CONFIG_WIFI_LOAD_DRIVER_WHEN_KERNEL_BOOTUP=y
-CONFIG_AP6XXX=y
-CONFIG_RTL8188EU=y
-CONFIG_MWIFIEX=y
-CONFIG_MWIFIEX_SDIO=y
+CONFIG_WIFI_BUILD_MODULE=y
+CONFIG_AP6XXX=m
+CONFIG_RTL8188EU=m
+CONFIG_RTL8723DS=m
+CONFIG_MWIFIEX=m
+CONFIG_MWIFIEX_SDIO=m
 # CONFIG_INPUT_MOUSEDEV is not set
 CONFIG_INPUT_JOYDEV=y
 CONFIG_INPUT_EVDEV=y
-- 
2.35.3

